h/browser/chrome/help/index.html

```diff
@@ -105,6 +105,13 @@ <h1 class="help-item__heading">We’re sorry, Hypothesis doesn’t work on this
         <img class="help-item__icon" src="sad-annotation.svg" />
         <h1 class="help-item__heading">We’re sorry, Hypothesis doesn't work on this site yet.</h1>
       </section>
+      <section id="other-error" class="help-item">
+        <img class="help-item__icon" src="sad-annotation.svg" />
+        <p class="center">
+          There was a problem loading Hypothesis into this page:
```

Comment:
I agree about the `<h1>` and adding a note that the error has been reported to us. As far as copy for errors go, I'll have a re-think but I do think it is important to keep the message simple and short as well as humble. Making the user read long sentences after you've screwed up is annoying. Also, if the user relays that message onto someone else (eg. Jeremy, Jon) they are less likely to lose the gist if the message is short.

These errors are reported to Sentry.